File: Front/src/Models/Lote.ts
import { Evento } from "./Eventos/Evento";

export interface Lote {
  id: number;
  nome: string;
  preco: number;
  dataInicio?: Date;
  dataFim?: Date;
  quantidade: number;
  eventoId: number;
  evento: Evento;
}

File: Back/src/ProEventos.Domain/Entities/Lote.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace ProEventos.Domain.Entities
{
    public class Lote : BaseEntity
    {
        public string Nome { get; set; }
        public decimal Preco { get; set; }
        public DateTime DataIncio { get; set; }
        public DateTime DataFim { get; set; }
        public int Quantidade { get; set; }               
        public int EventoId { get; set; }
        public Evento Evento { get; set; }
    }
}